### ISSUE TITLE
How Do You Remove This?

### ISSUE THREAD

**Issue by gamingvirtue** [4lex4nder/ReshadeEffectShaderToggler#43]
Literally as stated in the title. I've removed the addon files, all of the associated shaders, and completely wiped out ReShade and reinstalled it. And yet, my game remains plagued by this film grain shader. I genuinely have no idea what I'm doing wrong at this point.

**Comment by 4lex4nder** [comment]
1. Close the game
2. Remove ReshadeEffectShaderToggler.addon64 (and ReshadeEffectShaderToggler.addon32 if present)

That's it

**Comment by gamingvirtue** [comment]
Then I'm at a loss. Because I did that, manually stripped out all of the files ReShade touched (shaders, dxgi.dll, etc) and reinstalled it, and only reinstalling the entire game from scratch fixed this issue.

Installing this addon was also the only thing I did prior to the weird "VHS artifacts shader when the UI's not on screen" thing started happening, so that's all I can guess was causing it, since I had no such shader enabled, and even erased all of them in an attempt to fix the issue.
